flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/SourceContextWatermarkOutputAdapter.java

```diff
@@ -42,4 +42,9 @@ public void emitWatermark(Watermark watermark) {
     public void markIdle() {
         sourceContext.markAsTemporarilyIdle();
     }
+
+    @Override
+    public void markActive() {
+        // will be set active with next watermark
```

Comment:
Do we need some kind of follow-up to implement this?

Reply:
This is the old interface; so I'd like to avoid it.